apps/web/src/screens/HomePage/sections/SIGs/SIGDetailPane.tsx

```diff
@@ -61,18 +50,12 @@ const SIGWebsite = styled.a`
     background: #2d9cdb;
     color: #fff;
   }
-`;
 
-/*
-const SIGDiscord: any = styled(SIGButton)`
-  background: #1166bb;
-  color: white;
-  &:hover {
-    color: white;
-    background: #0055aa;
+  &:hover #child{
+    transition: filter 0.17s ease-in;
+    filter: invert(100%) sepia(100%) saturate(0%) hue-rotate(230deg) brightness(101%) contrast(102%);
```

Comment:
move these properties to a different styled component, specifically `ExternalIconWebsite`

Reply:
Functionality is lost when I do this with my current implementation.